Sources/WalletConnectSign/Engine/Common/SessionEngine.swift

```diff
@@ -281,3 +281,12 @@ private extension SessionEngine {
         onEventReceived?(topic, event.publicRepresentation(), payload.request.chainId)
     }
 }
+
+extension SessionEngine.Errors: LocalizedError {
+    var errorDescription: String? {
```

Comment:
is it better to use WalletConnectError? we have `noSessionMatchingTopic` for example